cypress/integration/partnerProfile.spec.js

```diff
@@ -0,0 +1,63 @@
+describe("/:partner_id", () => {
+  before(() => {
+    cy.visit("gagosian-gallery")
+  })
+
+  it("renders metadata", () => {
+    cy.title().should(
+      "eq",
+      "Gagosian | Artists, Art for Sale, and Contact Info | Artsy"
+    )
+    cy.get("meta[name='description']")
+      .should("have.attr", "content")
+      .and("eq", "Explore Artists, Artworks, and Shows from Gagosian on Artsy")
+  })
+
+  it("renders page content", () => {
+    cy.get("h1").should("contain", "Gagosian")
+  })
+
+  context("mobile", () => {
+    before(() => {
+      cy.viewport("iphone-x")
+      cy.visit("gagosian-gallery")
+    })
+
+    it("renders metadata", () => {
+      cy.title().should(
+        "eq",
+        "Gagosian | Artists, Art for Sale, and Contact Info | Artsy"
+      )
+      cy.get("meta[name='description']")
+        .should("have.attr", "content")
+        .and(
+          "eq",
+          "Explore Artists, Artworks, and Shows from Gagosian on Artsy"
+        )
+    })
+
+    it("renders page content", () => {
+      cy.get("h1").should("contain", "Gagosian")
+    })
+
+    it("shows the list of shows", () => {
+      cy.visit("gagosian-gallery/shows")
+      cy.contains("Current Shows")
+    })
+    it("shows partner articles", () => {
+      cy.visit("gagosian-gallery/articles")
+      cy.contains("Articles")
+    })
+
+    xit("does not show contact information for non-active partner", () => {
+      // TODO: cy.visit(":partner_id/contact")
```

Comment:
Intentionally pended, yes?

Reply:
Yeah, I'm not sure what a good inactive partner to reference here is-- do you know one we could use?

Reply:
I've been trying a few different Looker queries but coming up empty for a good candidate.

Question tho: the test as written, would it actually distinguish between say a churned partner and a regular 404 for a non-existent partner? (I think that error message is the 404 one)

Reply:
The 404 is at the `/:partner_id/contact` route, we'd still be expecting the `/:partner_id` to 200 (I think? It was asserted without trying the parent route).

Reply:
Happy to merge and come back to this. (Tbh this maybe even seems kind of edge-casey for this test suite?)